packages/core/src/BaseRadio/BaseRadio.styles.tsx

```diff
@@ -29,4 +37,41 @@ export const { staticClasses, useClasses } = createClasses("HvBaseRadio", {
     },
   },
   icon: {},
+  checked: {
+    "& svg": {
+      border: `1px solid ${theme.colors.secondary}`,
+      backgroundColor: theme.colors.secondary,
+      color: theme.colors.atmo2,
+    },
+    "&$semantic": {
+      "& svg": {
+        border: `1px solid ${theme.colors.base_dark}`,
+        backgroundColor: theme.colors.base_light,
+        color: theme.colors.base_dark,
+      },
+    },
+    "&$disabled": {
+      "& svg": {
+        border: `1px solid ${theme.colors.secondary_60}`,
+        backgroundColor: theme.colors.secondary_60,
+        color: theme.colors.atmo3,
+      },
+    },
+  },
+  semantic: {
+    "& svg": {
+      border: `1px solid ${theme.colors.base_dark}`,
+      backgroundColor: theme.colors.base_light,
+    },
+  },
+  pagination: {
```

Comment:
I think these styles should be passed through the `Dot Pagination` component , maybe through the `icon` class , I'm not sure. If it's possible it would reduce the need to add the new `pagination` prop, but maybe I'm missing something tell me what you think